### DISCUSSION TITLE
what microphone was used in the sensor-community-example-config?

### DISCUSSION THREAD

**Discussion by pe0ter** [stas-sl/esphome-sound-level-meter#23] [Q&A] [answered]
I am using an ICS43434 microphone connected to an ESP-WROOM-32 board and the sensor.community-example-config.yaml script works great but when I compare the measurement values to two of my sensor.community dnms microphones, one using an ICS43434 mic and one with an IM72D128 mic I notice the measurements of the ESP32 are kind of compressed. Same shape, higher low-value, slightly lower high value. I used my 94dB calibrator on the ESP32 mic and it showed 94.27dB. 
The sos filter values you are using in this example script, what microphone brand/type do they belong to?

**Comment by stas-sl** [comment, marked as answer]
In the `community-example-config.yaml` there is no microphone specific filter, there is just A-weighting filter that works with any microphone. 

You can add microphone specific filters before A-weighting, see README on how to do this. I personally use INMP441 microphone, so those coefficients in README are for this mic (as noted in comments), but there are also coefficients for ICS43434 mic [here](https://github.com/stas-sl/esphome-sound-level-meter/discussions/13). Though I haven't tried it myself.

I don't remember exactly but I believe DNMS code doesn't apply any mic specific equalization at all. It has only A-weighting.

94 dB vs 94.27 dB seems to be pretty close?
